cli/query/query.go

```diff
@@ -301,8 +303,10 @@ func queryVoter(ctx *cli.Context) error {
 	if st.VoteTo != nil {
 		voted = fmt.Sprintf("%s (%s)", hex.EncodeToString(st.VoteTo.Bytes()), address.Uint160ToString(st.VoteTo.GetScriptHash()))
 	}
-	fmt.Fprintf(ctx.App.Writer, "\tVoted: %s\n", voted)
-	fmt.Fprintf(ctx.App.Writer, "\tAmount : %s\n", fixedn.ToString(&st.Balance, int(dec)))
-	fmt.Fprintf(ctx.App.Writer, "\tBlock: %d\n", st.BalanceHeight)
```

Comment:
Why changing this? It's a pretty direct write that doesn't require additional buffering.

Reply:
Removed.